build-infrastructure/release-pipeline-stack.yml

```diff
@@ -401,9 +401,9 @@ Resources:
       Description: CodeBuild project to build the ECS Agent Docker image tarball and RPM
       Environment:
         ComputeType: BUILD_GENERAL1_SMALL
-        Image: aws/codebuild/amazonlinux2-x86_64-standard:3.0
+        Image: aws/codebuild/amazonlinux2-x86_64-standard:5.0
```

Comment:
do we want/need to update any of our other image versions?  Can you explain this version bump?
(same question for 2.0 -> 3.0 below)

Reply:
we don't need to bump any of them, this is just for housekeeping more than anything, we did the same for the pr-build jobs in the previous PR: https://github.com/aws/amazon-ecs-agent/pull/3933